modules/sharedaddy.php

```diff
@@ -20,4 +20,9 @@
 
 function sharedaddy_loaded() {
 	Jetpack::enable_module_configurable( __FILE__ );
+	add_filter( 'jetpack_module_configuration_url_sharedaddy', 'jetpack_sharedaddy_configuration_url' );
+}
+
+function jetpack_sharedaddy_configuration_url() {
+	return admin_url( 'admin.php?page=jetpack#/settings?term=social%20buttons' );
```

Comment:
This does not seem ideal. We direct folks from the old settings page to the new settings page, where we have no configuration settings but instead a link to another page (either on WordPress.com or under Settings > Sharing in wp-admin). Can't we direct folks to the final destination right away?

Reply:
I suppose since the activate/deactivate links are already available in the module list, we could send them right to the settings page for sharing buttons.

Reply:
Fixed in b31e142